middleware/cmd/middleware/main.go

```diff
@@ -0,0 +1,35 @@
+package main
```

Comment:
How about a short description of what the tool compiled from this `package main` does? One line is fine. :)

Reply:
Thanks, but perhaps something that describes what the middleware actually does would be useful?

E.g
```
// Middleware is a service that receives requests from the App and pipes it through information to other backends.
```